dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/uploadconfig/elements/UploadCategoryBuilder.kt

```diff
@@ -66,6 +66,7 @@ data class UploadCategoryBuilder(
     /**
      * Adds a new cell to this section
      */
+    @Suppress("LongParameterList")
```

Comment:
Might it make sense to bundle the parameters to a class? Something like CellSpecification or something like that?

Reply:
I see, you removed it again.

Reply:
In general I agree.
However this whole devtools thing is already sooo deeply nested and really hard to understand for "strangers", that I thought that the reader might be happy in this case, that s/he does not need to strg-click another class here to know the inputs😆 

What do you think?

Reply:
I did not have the pleasure with the new toolbox yet. But as long as the structure would only be used here I'm absolutely fine with keeping it as is.